nbodykit/filter.py

```diff
@@ -0,0 +1,50 @@
+from nbodykit.base.mesh import MeshFilter
+
+import numpy
+
+class TopHat(MeshFilter):
```

Comment:
FourierSpaceTopHat? KSpaceTopHat? ...we should probably differentiate from the real-space equivalent

Reply:
Not sure yet. Probably filter.fourier.TopHat?

Reply:
We can always convert the class argument to an argument to support configuration space convolutions.